usage/conf/routes

```diff
@@ -1,5 +1,9 @@
 GET     /                                               controllers.UsageApi.index
 
+GET     /usage/image/:mediaId                           controllers.UsageApi.forMedia(mediaId: String)
+GET     /usage/article/:articleId                       controllers.UsageApi.forArticle(articleId: String)
+GET     /usage/date/:date                               controllers.UsageApi.forDate(date: String)
```

Comment:
THis could probably be an aggregation of CAPI vs this API unless we want print in it?